packages/p2p-v2/src/hooks/useCountdownTimer.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Custom hook for a countdown timer.
+ * @param {number} initialMilliseconds - The initial number of seconds for the countdown timer.
+ * @param {number} interval - The interval in milliseconds at which the timer will update.
+ * @returns {number} - The current value of the countdown timer in seconds.
+ */
+const useCountdownTimer = (initialMilliseconds: number, interval = 1000) => {
```

Comment:
can we create test cases for these hooks as well

Reply:
why dont we use useCountdown from usehooks ?

Reply:
ah.. yeah..  let me try with that..